app/models/tv_series.rb

```diff
@@ -35,30 +35,23 @@ def self.parse_search_results(results)
   end
 
   def self.parse_results(result, show_id)
-    @show_id = show_id
-    @first_air_date = Date.parse(result[:first_air_date]).stamp("1/2/2001") if result[:first_air_date].present?
-    @last_air_date = Date.parse(result[:last_air_date]).stamp("1/2/2001") if result[:last_air_date].present?
-    @show_name = result[:name]
-    @backdrop_path = result[:backdrop_path]
-    @poster_path = result[:poster_path]
-    @number_of_episodes = result[:number_of_episodes]
-    @number_of_seasons = result[:number_of_seasons]
-    @overview = result[:overview]
-    @seasons = (1..(result[:number_of_seasons])).to_a if result[:number_of_seasons].present?
-    @actors = TVCast.parse_results(result[:credits][:cast])
```

Comment:
I reconciled these instance variables with the html pages and we're safe to remove them. The view can access all of these values from the `@series` variable defined in the controller.